Composer/packages/ui-plugins/schema-editor/src/formSchema.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { UISchema } from '@bfc/extension';
+import { SDKKinds } from '@bfc/shared';
+
+import { AdaptiveDialogField } from './AdaptiveDialogField';
+
+const formSchema: UISchema = {
+  [SDKKinds.AdaptiveDialog]: {
+    field: AdaptiveDialogField,
```

Comment:
So the gist of this plugin is that it adds a "virtual" field to the adaptive dialog form? Is that right?

I'm wondering if there can be someway of extending the form schema to allow something like this. Do you think that would make this better?

Reply:
Yea I think it would be easier if we could just append the field to the bottom of the form. I'm not sure what would be the best way to represent that in the form schema would be and how it would be implemented. I guess we could handle it in the`SchemaField` component.